scripts/image-builder/build.sh

```diff
@@ -323,6 +324,10 @@ fi
 createrepo microshift-local >/dev/null
 open_repo_permissions microshift-local
 
+# Determine the version of microshift we have, for use in blueprint templates later.
+MICROSHIFT_VERSION=$(rpm -q --queryformat '%{version}' "${MICROSHIFT_RPM_SOURCE}/RPMS/noarch/microshift-release-info*.rpm")
```

Comment:
This code may not work when RPMs are downloaded from a remote site because it assumes `RPMS/noarch` suffix exists.

We should instead use `find | tail -1` to discover the last version of the release-info. Note that `tail -1` is required because even in devenv, the local repo may contain multiple copies of RPMs and we need the last one.